src/fides/api/ops/service/connectors/saas_connector.py

```diff
@@ -404,6 +404,51 @@ def mask_data(
         self.unset_connector_state()
         return rows_updated
 
+    def run_consent_request(
+        self,
+        node: TraversalNode,
+        policy: Policy,
+        privacy_request: PrivacyRequest,
+        identity_data: Dict[str, Any],
+    ) -> bool:
```

Comment:
This is where Saas connector experts would step in to start working on how to make a consent request to a third party. This method doesn't currently make a request.